.github/workflows/build-and-deploy-notifications.yml

```diff
@@ -0,0 +1,64 @@
+name: Build and Deploy
+
+on:
+  push:
+    branches:
+    - main
+    - deploy-resources
+  pull_request:
+    branches:
+    - main
+
+  workflow_dispatch:
+
+concurrency:
```

Comment:
can we move this concurrency group to the build job.  on other projects it's caused the terraform statefile to get locked